### DISCUSSION TITLE
What are correct settings to select?

### DISCUSSION THREAD

**Discussion by LepaBrenaa** [maltejur/discord-screenaudio#109] [Q&A] [answered]
How should my Input and Output settings look like when screen-sharing?

Buddies keep telling me that the sound in games works just fine, but they hear me twice. Once though discord normally, and once through screen-share.

Does `discord-screeanaudio-virtmic` have to be explicitly set as `Input Device``:
![image](https://user-images.githubusercontent.com/48409293/219974112-9c9ff374-92f7-4186-8692-0d8b00a7d357.png)

or can I select (in my case) `SteelSeries Arctis 7 Chat`?:
![image](https://user-images.githubusercontent.com/48409293/219974139-3ab1e7cf-5792-489e-ad46-5a7fe2852266.png)

When wanting to screen-share, I typically select the game in `Audio Source`, and then choose the Game window to stream it:
![image](https://user-images.githubusercontent.com/48409293/219974179-baf72be6-bd8c-45f9-9c2b-38c1869c1f71.png)
![image](https://user-images.githubusercontent.com/48409293/219974188-7ae2e535-8815-4108-83d9-533e302fcfb4.png)

Am I doing something wrong?

**Comment by maltejur** [comment, marked as answer]
You should just select your normal microphone, not the `discord-screenaudio-virtmic`. Apart from that, you are doing everything correctly.

**Comment by bytepandabun** [comment]
you was a life saver brother.
